Project.toml

```diff
@@ -9,6 +9,7 @@ IfElse = "615f187c-cbe4-4ef1-ba3b-2fcf58d6d173"
 LoopVectorization = "bdcacae8-1622-11e9-2a5c-532679323890"
 ManualMemory = "d125e4d3-2237-4719-b19c-fa641b8a4667"
 PolyesterWeave = "1d0040c9-8b98-4ee7-8388-3f51789ca0ad"
+Requires = "ae029012-a4dd-5104-9daa-d747884805df"
```

Comment:
Also needs a compat bounds.
This caused the test failures.

Reply:
Oh, right :facepalm: Sorry. Should be fixed in f0536a4